tools/pipeline/artifact/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+revng_add_executable(revng-artifact Main.cpp)
+
+target_link_libraries(revng-artifact revngPipeline revngPipes revngRecompile
+                      revngEarlyFunctionAnalysis)
```

Comment:
Why `revngRecompile` and `revngEarlyFunctionAnalysis`? I guess we could just have `revngPipeline` here.